pkg/cluster/k8sres.go

```diff
@@ -1571,6 +1571,14 @@ func (c *Cluster) generateLogicalBackupPodEnvVars() []v1.EnvVar {
 			Name:  "LOGICAL_BACKUP_S3_BUCKET",
 			Value: c.OpConfig.LogicalBackup.LogicalBackupS3Bucket,
 		},
+		{
+			Name:  "LOGICAL_BACKUP_S3_ENDPOINT",
+			Value: c.OpConfig.LogicalBackup.LogicalBackupS3Endpoint,
+		},
+		{
+			Name:  "AWS_SSE",
```

Comment:
any reason why this env variable doesn't have the `LOGICAL_BACKUP_` prefix?

Reply:
no, that was my mistake. Dump.sh is already expecting LOGICAL_BACKUP_S3_SSE. I've fixed that.